src/components/PdfReaderComponent.js

```diff
@@ -163,6 +162,7 @@ class PdfReaderComponent {
     await this.#setCanvasSize();
 
     this.#addVisibilityListenersToPages();
+    this.components.loader.classList.add("hidden");
```

Comment:
This should not be here. Function setupPages should not know about some loader html element. This is not the responsibility of setupPages method. Cleaner solution would be to create 'onPdfLoaded' event.